<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateSaveGameConfigWindow</class>
 <widget class="QWidget" name="CreateSaveGameConfigWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateSaveGameConfigWindow</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="executableLocationLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseExeButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Executable Path</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Savegame Location</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="savegameLocationLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browseSaveGameLocation">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="confirmButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>confirmButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateSaveGameConfigWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateSaveGameConfigWindow</receiver>
   <slot>OkClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseExeButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateSaveGameConfigWindow</receiver>
   <slot>BrowseExecutable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseSaveGameLocation</sender>
   <signal>clicked()</signal>
   <receiver>CreateSaveGameConfigWindow</receiver>
   <slot>BrowseSavegameLocation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OkClicked()</slot>
  <slot>BrowseExecutable()</slot>
  <slot>BrowseSavegameLocation()</slot>
 </slots>
</ui>
